### PR TITLE
Upload a datestamped result set to S3

### PR DESCRIPTION
**Pull request by johnhawkinson**
We could probably architect this better, but this should do for now.

### CODE REVIEW

**Review by carloshgaray — APPROVED** [review]
sgtm let's have a sync late next week on a design session

File: main.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ async function execute() {
 			};
 
 			// Uploading files to the bucket
-			return await new Promise((resolve, reject) => {
+			const results = await new Promise((resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (livgust) [reply main.js:87]: @carloshgaray he added a second promise so it'll write to data.json and also the timestamped file.
@johnhawkinson Is there a reason you're returning the results of the initial promise? I don't think we actually care about the return value here.

**Review by livgust — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (johnhawkinson) [reply main.js:87]: I was just trying to make the most surgical change possible without affecting anything else.
You were returning the data.json result before, so I wanted to keep that.

[the remark this answers is not in the document]
Comment (livgust) [reply main.js:87]: LGTM. I'll test, merge, and upload to Lambda when the kids go to bed. Then I'll look into making the S3 bucket public so people can use these files for data analytics and such.

**Review by livgust — APPROVED** [review]

**Comment by livgust** [comment]
Merging to master, then going to zip to Lambda for a test run

**Comment by livgust** [comment]
Looks good but file name has a colon: data-2021-02-13T03:30Z.json

**Comment by johnhawkinson** [comment]
Bah humbug. Colons aren't illegal in most OS's filenames, but they are a really bad idea.
I assume you're saying we should change it…

**Pull request merged**